charts/ceph-csi-rbd/values.yaml

```diff
@@ -128,13 +128,7 @@ provisioner:
   name: provisioner
   replicaCount: 3
   strategy:
-    # RollingUpdate strategy replaces old pods with new ones gradually,
-    # without incurring downtime.
-    type: RollingUpdate
-    rollingUpdate:
-      # maxUnavailable is the maximum number of pods that can be
-      # unavailable during the update process.
-      maxUnavailable: 50%
+    type: Recreate
```

Comment:
What is the reason to change this? The comments from commit ea088d40be are very descriptive, not sure this is an improvement?

Reply:
Oh! sorry, this is just a workaround. Currently, when we set Strategy type to Rollingupdate, the Patch command is working differently, i.e while the old pod is still Running a new pod is getting created and waiting in the pending state, the old pod is not getting terminated (unless someone manually kill it) and the new pod is waiting for old one to get terminated, obviously, the status of pod replica is set to 2 out of which 1 is not ready. This scenario is leading to other failures in the CI, such as waiting for provisioner 1/2 state.

That said, when the `Rollingupdate: {} ` attribute us set initially k8s cannot give us the provision to patch/apply it to `null`, only manual edit is allowed.

That all said, this is just an experiment PR for now. Which is anyway not working as expected.

Reply:
> That all said, this is just an experiment PR for now. Which is anyway not working as expected.

Ok! In that case, I'll mark this as a **Draft** for the time being.